From c7ec57613d277934a197d501277c208b6cfcf9ff Mon Sep 17 00:00:00 2001
From: Poovendhan Selvaraj <quic_poovendh@quicinc.com>
Date: Fri, 6 Oct 2023 12:18:19 +0530
Subject: [PATCH] crypto: testmgr: Disable uneven data len AES CTR tests

This patch disables the uneven data len testcases by default
in ipq platforms.
(cherry picked from commit 0282f440bc6af56408800d09a6dc1ba08eab6189)

Change-Id: Ie0edec4a03949ac4b4173ea25d7df3a5d9d00bfa
Signed-off-by: Poovendhan Selvaraj <quic_poovendh@quicinc.com>
---
 crypto/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/crypto/Kconfig b/crypto/Kconfig
index b99d34d6e7d1..79dbaef3145a 100644
--- a/crypto/Kconfig
+++ b/crypto/Kconfig
@@ -229,6 +229,13 @@ config CRYPTO_TEST
 	help
 	  Quick & dirty crypto test module.
 
+config CRYPTO_NO_AES_CTR_UNEVEN_DATA_LEN_SUPPORT
+	bool "Disable support for uneven data length test cases"
+	default y
+	depends on CRYPTO_TEST
+	help
+	  Disable support for uneven data length cases
+
 config CRYPTO_NO_AES_XTS_ZERO_KEY_SUPPORT
 	bool "Disable support for zero key test cases"
 	default y
-- 
2.34.1

